app/partials/trade-summary.jade

```diff
@@ -4,9 +4,9 @@ div(data-preflight-tag="TradeSummary")
   div(ng-if="formattedTrade.tx")
     label.em-500.flex-start.pts.mtl(ng-hide="isPendingBankTransfer && isPendingBankTransfer()" translate="{{isKYC || needsKyc() ? 'DETAILS' : 'ORDER_DETAILS'}}")
     label.em-500.flex-start.pts.mtl(ng-show="isPendingBankTransfer && isPendingBankTransfer()" translate="BANK_ORDER_DETAILS")
-    ul.em-300.bg-grey.mtm.mbml
-      li.pam(ng-repeat="(prop, val) in formattedTrade.tx track by $index" ng-if="val")
-        span.em-500.colon.phm(translate="{{prop}}")
-        span.em-400(ng-bind="val")
-      li.phm.pbm.parens.type-sm.mhm(ng-show="isPendingBankTransfer && isPendingBankTransfer()" translate="ORDER_ID_REFERENCE") 
-p(ng-show="isPendingBankTransfer && isPendingBankTransfer()" translate="BANK_TRANSFER_NEXT_STEPS" translate-values="formattedTrade.values")
\ No newline at end of file
+    ul.em-300.bg-light-blue.border-top(ng-class="{'bank-transfer': isPendingBankTransfer && isPendingBankTransfer()}")
```

Comment:
The class 'bank-transfer' will change the style of the summary. See https://github.com/blockchain/My-Wallet-V3-Frontend/pull/867/files#diff-1eb8d7b43d1798135358050888f1357dR417